Gir.toml

```diff
@@ -356,6 +356,28 @@ builders = [
 
 ignore = [
     "Gtk.Activatable",
+    # base class
+    "Gtk.WidgetBuilder",
+    # deprecated
+    "Gtk.ArrowBuilder",
+    # base class
+    "Gtk.MiscBuilder",
+    # base class
```

Comment:
We can easily detect this case

```xml
    <class name="Misc"
           c:symbol-prefix="misc"
           c:type="GtkMisc"
           parent="Widget"
           abstract="1"
           glib:type-name="GtkMisc"
           glib:get-type="gtk_misc_get_type"
           glib:type-struct="MiscClass">
```

Note the `abstract=1` here. That could be parsed by gir

Reply:
Oh interesting! I'll add it in gir then.